op-e2e/setup.go

```diff
@@ -624,12 +628,19 @@ func (cfg SystemConfig) Start(t *testing.T, _opts ...SystemConfigOption) (*Syste
 		return nil, err
 	}
 
-	for name := range cfg.Nodes {
-		if name == RoleL1 {
-			return nil, fmt.Errorf("node name %s is reserved for L1 node", RoleL1)
-		}
+	// Ordered such that the Sequencer is initialized first. Setup this way so that
+	// the `RollupSequencerHTTP` GethOption can be supplied to any sentry nodes.
+	l2Nodes := []string{RoleSeq, RoleVerif}
+	for _, name := range l2Nodes {
```

Comment:
If someone added a node to the `cfg.Nodes` then that nodes will not be started anymore, seems like a bug.
Maybe you can set `l2Nodes := maps.Keys(cfg.Nodes)`, and then sort the `l2Nodes`, with a sort-func that puts the `RoleSeq` first?

Reply:
yea good call. Let me update